src/main/resources/application.properties

```diff
@@ -20,7 +20,7 @@ spring.http.multipart.max-file-size=1000MB
 
 ##sms details for CHO
 CHOSmsTemplate= CHO UPTSU SMS
-sendSMSUrl= <Enter SMS URL here>
+sendSMSUrl= http://10.208.122.38:8080/commonapi-v1.0/sms/sendSMS
```

Comment:
@swetatech24 You should not put the actual URLs in properties file. Replace this with the placeholder. We had discussed this multiple times.